### DISCUSSION TITLE
Welcome to chipKIT-core Discussions!

### DISCUSSION THREAD

**Discussion by EmbeddedMan** [chipKIT32/chipKIT-core#464] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EmbeddedMan** [comment]
Sorry, this is just the template GH discussion topic. I don't know enough (yet) about GH discussions to know how best to taylor this to our use. We'll get there together.

**Reply by majenkotech** [reply]
Awesome. I'll link to the root of the discussions from the banner at the top of the old forums.

**Reply by cb88** [reply]
What is the plan with chipkit.net? Redirect it to a clean github.io landing page perhaps? The site and forum there are pretty broken unfortunately, I bet that is a decent amount of annoyance to fix. It would be nice to keep it as an archive of past conversations but it would be better for it to just disappear than to stay broken.

**Reply by majenkotech** [reply]
The plan is a completely new website. It's about 95% done. Forums are here, with an archive of the old posts that can be searched. The WiKi is re-written and the blogs ported, and all re-hashed using Markdown (my favourite layout system). And all with a single common sign-on, which can use Github as the backend authentication (for those that need to log in, which is only bloggers and WiKi editors at the moment).
